src/sympc/protocol/spdz/spdz.py

```diff
@@ -76,6 +150,9 @@ def mul_parties(
     :rtype: ShareTensor (in our case ShareTensorPointer)
     """
 
+    crypto_store = session.crypto_store
+    key_store = f"beaver_mul_{tuple(eps.shape)}_{tuple(delta.shape)}"
```

Comment:
This `key_store` sounds a bit hacky
I understand it is the equivalent of this: https://github.com/OpenMined/PySyft/blob/syft_0.2.x/syft/frameworks/torch/mpc/spdz.py#L84
So I wouldn't use strings I think!

Reply:
Hmm - yep, but I was thinking that we need a way to "register" custom "setters" and "getters" for primitives? - I can try something different - will update the PR and ping you when is ready :D